crates/holochain/src/conductor/space.rs

```diff
@@ -625,20 +625,20 @@ impl Spaces {
         dna_hash: &DnaHash,
         request_validation_receipt: bool,
         countersigning_session: bool,
-        ops: Vec<holochain_types::dht_op::DhtOp>,
+        ops: Vec<DhtOp>,
     ) -> ConductorResult<()> {
-        use futures::StreamExt;
-        let ops = futures::stream::iter(ops.into_iter().map(|op| {
-            let hash = DhtOpHash::with_data_sync(&op);
-            (hash, op)
-        }))
-        .collect()
-        .await;
-
         // If this is a countersigning session then
         // send it to the countersigning workflow otherwise
         // send it to the incoming ops workflow.
         if countersigning_session {
+            use futures::StreamExt;
```

Comment:
It pairs with the change in the top of the workflow. We were hashing twice, once before and once right before saving in the incoming ops workflow. We now just hash once. Because the code branches here and in the interest of maintaining existing behavior I have moved the code that is no longer needed by the incoming ops workflow inside this branch